### ISSUE TITLE
info: what's the main purpose of this repo?

### ISSUE THREAD

**Issue by iraunit** [Ajayjeena/Crawl0#1]
Could you please explain me the main purpose of this, also will it only discover the links or the page content as well?

**Comment by Ajayjeena** [comment]
Yeah sure, for time being it will be used just to get all the links in a website and then discover another links from those links which are still undiscovered.
